x-pack/filebeat/module/cisco/meraki/test/generated.log-expected.json

```diff
@@ -60,8 +60,8 @@
             "appliance"
         ],
         "related.ip": [
-            "10.15.16.212",
-            "10.102.218.31"
+            "10.102.218.31",
```

Comment:
I would recommend when generating test data that you specify the module and fileset, I get this issue from time to time as well.
Depending on how you run your expected commands it should be something like:
`TESTING_FILEBEAT_MODULES=cisco TESTING_FILEBEAT_FILESETS=asa`

Reply:
Good to know...I had been using:
GENERATE=1 TESTING_FILEBEAT_MODULES=cisco mage pythonIntegTest

Which helps explain why it was updating the meraki stuff even though I hadn't changed anything having to do with Meraki.